pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -573,6 +574,11 @@ func (StandardSecurityPolicyEnforcer) EnforceScratchUnmountPolicy(string) error
 	return nil
 }
 
+// Stub. We are deprecating the standard enforcer.
+func (StandardSecurityPolicyEnforcer) GetUserInfo(spec *oci.Spec) (IDName, []IDName, string, error) {
```

Comment:
nit: `// Deprecated: Do not use [StandardSecurityPolicyEnforcer]`, so the linter catches and warns about use

Reply:
Noted.